### PR TITLE
Dont expose Powered-By: Prestashop 

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The header could be misused by crawlers in case PrestaShop has know security issues in FO.
| Type?         | improvement
| Category?     | FO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | Read the header

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11362)

### CODE REVIEW

**Review by Shine-neko — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Nice catch @MathiasReker Really good idea :)
(Don't know about this header ^^)
I need to have a talk with @toutantic  about this PR on tomorrow, someone whispered in my ear that it was important ><

**Comment by MathiasReker** [comment]
@PierreRambaud Should I change branch to 1.7.5.x?

**Comment by PierreRambaud** [comment]
@MathiasReker Still waiting for feedback from someone in other team.
But as I know, there are many others way to know if the website is powered by PrestaShop or not, it's really minor since the version is not pushed in the header :)

**Comment by MathiasReker** [comment]
@PierreRambaud I agree with you, there are many other ways to know if the website is powered by PrestaShop. I don't see this as critical at all, just an improvement. :-)

**Comment by PierreRambaud** [comment]
Hi @MathiasReker I'm sorry, we are still in discussion about this improvement :)

**Comment by rdy4ever** [comment]
Well, there are a lot of other ways to know the site runs Prestashop. For exemple, you can test the existence of known Prestashop paths, like modules. Let's say a bad bot tests for the existence of this path, available in most Prestashop installations: `/modules/ps_imageslider/js/responsiveslides.min.js`. It will know for sure the site runs PS.
Also, take a look at the cookies:
image

In fact, it is very hard (if not impossible) to hide any open source software from hackers. You may hide it from the eyes of regular users, but that's about it.

TLDR: you can't hide it. Read **[this article](https://www.drupal.org/node/766404)** about 'hiding' another open source software, Drupal.

> "Security by obscurity is no security. It gives a false impression of being "safe" when you are only hiding vulnerabilities behind a smokescreen that any attacker that posed any real threat would be able to see through."

**Comment by PierreRambaud** [comment]
Hehe I already know @rdy4ever that is why I want to merge this one :)

**Comment by PierreRambaud** [comment]
I merge this one, it does not need QA. Seems with @toutantic and other people of the PrestaShop team.
Thanks @MathiasReker for your contribution and @rdy4ever for the article, which will allow everyone to understand why this header is useless.

**Comment by MathiasReker** [comment]
Thanks @PierreRambaud
